lib/services.js

```diff
@@ -90,11 +91,27 @@ module.exports = function generateServices(app, options) {
     { encoding: 'utf-8' }
   );
 
+  var commonModelPrefix = 'LoopBack';
+  if (options.namespaceCommonModels) {
+    commonModelPrefix = options.ngModuleName + options.namespaceDelimiter;
+    if (options.namespaceDelimiter === '.') {
+      return 'throw new Error("Unsupported delimiter for ' +
+        'namespacing common models.");';
```

Comment:
Please mention the invalid delimiter value `.` in the error message to prevent user confusion.